vhdbuilder/packer/cleanup.sh

```diff
@@ -181,7 +181,58 @@ if [[ -n "${AZURE_RESOURCE_GROUP_NAME}" && "${DRY_RUN,,}" == "false" ]]; then
   set -x
 fi
 
-#clean up storage account created over a week ago
+# attempt to clean up Windows managed images and SIG image versions created over a week ago in SIG_GALLERY_NAME (except the production gallery)
+if [[ -n "${AZURE_RESOURCE_GROUP_NAME}" && "${DRY_RUN}" == "False" && "${SIG_GALLERY_NAME}" != "AKSWindows" ]]; then
```

Comment:
we may not run so many cleanup steps in `"${DRY_RUN}" == "False"` which means that it will be run only when we try to publish new Windows images. It will increase additional cost for us.

Reply:
yeah I commented about this below, thinking we can potentially create a separate pipeline variable to dictate this behavior and not complicate the behavior or dry-run